app/views/publishers/vacancies/_actions.html.slim

```diff
@@ -1,9 +1,9 @@
 .vacancy-actions
   - if vacancy.published?
-    = govuk_button_link_to t("buttons.copy_listing"), new_organisation_job_copy_path(vacancy.id), class: "govuk-button--secondary"
+    = govuk_button_link_to t("buttons.copy_listing"), new_organisation_job_copy_path(vacancy.id), class: "govuk-button--secondary govuk-!-margin-right-2"
```

Comment:
can you use the `govuk-button-group` thing so all the spacing is correct and you dont need the  repeated `govuk-!-margin-right-2` ?

Reply:
i think `.govuk-button-group` could replace `.vacancy-actions`